docs/cunumeric/source/comparison/_comparison_generator.py

```diff
@@ -62,12 +84,22 @@ def _import(mod, klass):
         return obj, ":obj:`{}.{{}}`".format(mod)
 
 
-def _section(header, mod_ext, other_lib, klass=None, exclude_mod=None):
+def _section(
+    header, mod_ext, other_lib, klass=None, exclude_mod=None, ufuncs=False
+):
     base_mod = "numpy" + mod_ext
     other_mod = other_lib + mod_ext
 
+    if ufuncs:
+        klass = None
+        exclude_mod = None
+
+    base_funcs = []
     base_obj, base_fmt = _import(base_mod, klass)
-    base_funcs = _get_functions(base_obj)
+    if ufuncs:
```

Comment:
I guess you could pass a getter as an argument, instead of choosing it on a boolean flag.

Reply:
Sure